gst/nnstreamer/nnstreamer_plugin_api_filter.h

```diff
@@ -249,6 +256,12 @@ get_accl_hw_type (const char * str);
 extern const char *
 get_accl_hw_str (const accl_hw key);
 
+/**
+ * @brief check if the given accelerator is supported
+ */
+extern int
+gst_tensor_filter_check_hw_support (const char * const *supported_hw_list, accl_hw hw);
```

Comment:
Usually, this is supposed to be probed in run-time (check if the hardware is really available)... Then, providing such an internal API is awkward.

Reply:
This API currently determines statically. Later, the implementation of this API will be updated to determine if the hardware is really available (dependent on the platform).

Reply:
I don't think we can provide some internal APIs for hardware detection. The responsibility should be on subplugins, which are supposed to be closer to layers of device drivers. Otherwise, we are breaking layers of platform architectures.

Reply:
Yes, that is right. Updated to directly use `g_strv_contains`.